lib/node_modules/@stdlib/blas/base/idamax/benchmark/fortran/benchmark.length.f

```diff
@@ -0,0 +1,217 @@
+    rate = dble( iterations ) / elapsed
+    ! ..
+    print '(A)', '  ---'
+    ! ..
+    write (str, '(I15)') iterations
+    tmp = adjustl( str )
+    print '(A,A)', '  iterations: ', trim( tmp )
+    ! ..
+    write (str, '(f0.9)') elapsed
+    tmp = adjustl( str )
+    print '(A,A)', '  elapsed: ', trim( tmp )
+    ! ..
+    write( str, '(f0.9)') rate
+    tmp = adjustl( str )
+    print '(A,A)', '  rate: ', trim( tmp )
+    ! ..
+    print '(A)', '  ...'
+  end subroutine print_results
+  ! ..
+  ! Runs a benchmark.
+  !
+  ! @param {integer} iterations - number of iterations
+  ! @param {integer} len - array length
+  ! @return {double} elapsed time in seconds
+  ! ..
+  double precision function benchmark( iterations, len )
+    ! ..
+    ! External functions:
+    interface
+      integer function idamax( N, dx, strideX )
+        integer :: strideX, N
+        double precision :: dx(*)
+      end function idamax
+    end interface
+    ! ..
+    ! Scalar arguments:
+    integer, intent(in) :: iterations, len
+    ! ..
+    ! Local scalars:
+    integer :: idx
+    double precision :: elapsed, r
+    double precision ::  t1, t2
+    integer :: i
+    ! ..
+    ! Local arrays:
+    double precision, allocatable :: x(:)
+    ! ..
+    ! Intrinsic functions:
+    intrinsic random_number, cpu_time
+    ! ..
+    ! Allocate arrays:
+    allocate( x(len) )
+    ! ..
+    do i = 1, len
+      call random_number( r )
+      x( i ) = ( r*2000.0 ) - 1000.0
+    end do
+    ! ..
+    idx = -1
+    call cpu_time( t1 )
+    ! ..
+    do i = 1, iterations
+      idx = idamax( len, x, 1 )
+      if ( idx /= idx ) then
+        print '(A)', 'should not return NaN'
+        exit
+      end if
+    end do
+    ! ..
+    call cpu_time( t2 )
+    ! ..
+    elapsed = t2 - t1
+    ! ..
+    if ( idx /= idx ) then
+      print '(A)', 'should not return NaN'
+    end if
+    ! ..
+    ! Deallocate arrays:
+    deallocate( x )
+    ! ..
+    benchmark = elapsed
+    return
+  end function benchmark
+  ! ..
+  ! Main execution sequence.
+  ! ..
+  subroutine main()
+    ! ..
+    ! Local variables:
+    character(len=999) :: str, tmp
+    double precision :: elapsed
+    integer :: i, j, len, count, iter
+    ! ..
+    ! Intrinsic functions:
+    intrinsic adjustl, trim
+    ! ..
+    call print_version()
+    count = 0
+    do i = min, max
+      len = 10**i
+      iter = iterations / 10**(i-1)
+      do j = 1, repeats
+        count = count + 1
+        ! ..
+        write (str, '(I15)') len
+        tmp = adjustl( str )
+        print '(A,A,A,A)', '# fortran::', name, ':len=', trim(tmp)
+        ! ..
+        elapsed = benchmark( iter, len )
+        ! ..
+        call print_results( iter, elapsed )
+        ! ..
+        write (str, '(I15)') count
+        tmp = adjustl( str )
+        print '(A,A,A)', 'ok ', trim( tmp ), ' benchmark finished'
+      end do
+    end do
+    call print_summary( count, count )
+  end subroutine main
+end program bench
```

Comment:
add new line at end of file

Reply:
Our current `.editorconfig` forbids trailing newlines for Fortran files.

While may change this in the near future (see @kgryte's comment on here:
https://github.com/stdlib-js/stdlib/commit/0244027e1e2c0ceb1cd8ae1808196c24fa77b142#r140900051), I suggest we do not change anything in this PR with respect to EOF newlines for Fortran files.